.changeset/gorgeous-timers-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-gitlab': minor
+---
+
+added test if user does not have name or profile picture,variable default to undefined
```

Comment:
```suggestion
Fixed bug that caused ingestion errors when a user's name or profile picture is undefined.
```

Reworded this slightly as this is what goes in the changelog for this package. The tests are a great addition but an internal thing to the package so no need mentioning that.